members/SOHG1fc662bc.yaml

```diff
@@ -0,0 +1,67 @@
+bioguide: SOHG1fc662bc
+contact_form:
+  method: post
+  action: ""
+  steps:
+    - visit: "https://governor.ohio.gov/wps/portal/gov/governor/contact"
+    - fill_in:
+        - name: First Name
+          selector: input#first-name
+          value: $NAME_FIRST
+          required: true
+        - name: Last Name
+          selector: input#last-name
+          value: $NAME_LAST
+          required: true
+        - name: Street
+          selector: input#street
+          value: $ADDRESS_STREET
+          required: true
+        - name: City
+          selector: input#city
+          value: $ADDRESS_CITY
+          required: true
+        - name: zip5
+          selector: input#zip-code
+          value: $ADDRESS_ZIP5
+          required: true
+        - name: Phone
+          selector: input#phone-number
+          value: $PHONE
+          required: true
+        - name: email
+          selector: input#txt-email
+          value: $EMAIL
+          required: true
+        - name: Subject
+          selector: input#subject
+          value: $SUBJECT
+          required: true
+        - name: Message
+          selector: textarea#txt-message
+          value: $MESSAGE
+          required: true
+    - select:
+        - name: Topic
+          selector: select#select-issue
+          value: $TOPIC
+          required: true
+          options:
+            General: General
+        - name: State
+          selector: select#select-state
+          value: $ADDRESS_STATE_FULL
+          required: true
+          options: Ohio
+    - click_on:
+        - value: Submit
+          selector: div.col-sm-12 button.margin-bottom-xs
+    - wait:
+        - value: 1
+    - find:
+        - selector: div.odx-message__success-4
```

Comment:
Have you tried submitting an empty form and made sure we don't find the text on this hidden success message element?

Reply:
Yeah. I did a few tests: with an empty form https://s3.amazonaws.com/yamltron-dev/screenshots/error/c011504d-7f5b-43e4-9a82-e11b79779e6e.png

and a form with some inputs filled out, https://s3.amazonaws.com/yamltron-dev/screenshots/error/4f848510-0417-485c-a03d-372098046d88.png

and the hidden messages deos not appear